packages/perseus/src/widgets/interactive-graphs/stateful-mafs-graph.tsx

```diff
@@ -127,7 +127,7 @@ export const StatefulMafsGraph = React.forwardRef<
 
     // If the graph is static, it always displays the correct answer. This is
     // standard behavior for Perseus widgets (e.g. compare the Radio widget).
-    if (props.static) {
+    if (props.static && props.correct) {
```

Comment:
This is the only place we use `correct` inside the interactive graph components.